### PR TITLE
Create new cv html page

### PR DESCRIPTION
**Pull request by ZHANGJialiHappy**
1. Create a new page.
2. Add a photo.

@zdlm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<style>
+.myDiv{
+    font-size:80%;
+}
+</style>
+
+</head>
+<body>
+
+<h1> <b>Jiali Zhang</b></h1>
```
Comment (ZHANGJialiHappy) [inline index.html]: You'd better not use <b>,  Make a style in the h1

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    font-size:80%;
+}
+</style>
+
+</head>
+<body>
+
+<h1> Jiali Zhang</h1>
+<img src="photo.jpg" alt="Jiali Zhang's photo" width="100" height="105" />
+<p><b>(+45) 50657766</b></p>
```
Comment (zdlm) [inline index.html]: Please don't use <b> to make the text bold

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<a href="https://github.com/ZHANGJialiHappy"> Github Link </a>
+<pre>I have lived in 3 countries of Europe for 12 years,
```
Comment (zdlm) [inline index.html]: Pre is not common html tag. Use <div> we make make the style in the future.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+<pre>I have lived in 3 countries of Europe for 12 years, 
+and now I settle down in Copenhagen with my families, 
+so I can speak German, English, Chinese and Danish. 
+I have several years working experience in Marketing 
+and Project Management. 
+</pre>
+
+<h3 style="color:blue;"> <b> MY OWN PROJECTS </b></h3>
+
+<p><b>Norsden Aps, Copenhagen</b> — Co-Founder</p>
```
Comment (zdlm) [inline index.html]: Same with <b>

File: index.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+<div class="myDiv"> more than an online shop and retail</div>
+<p> August 2020 - Present </p>
+
+<p><b>Wechat Official Account: Living in Denmark</b></p>
```
Comment (zdlm) [inline index.html]: And here <b>

File: index.html [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+<div class="myDiv">Introduce everything in Denmark, like laws, gourmet and so on</div>
+<p>December 2016 - Present</p>
+<ul>
```
Comment (zdlm) [inline index.html:34]: It is good to use ul and li

**Review by zdlm — APPROVED** [review]

**Review by ZHANGJialiHappy — COMMENTED** [review]
I corrected all

**Pull request merged**